src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/db
+spring.datasource.username=admin
+spring.datasource.password=admin
+
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database=postgresql
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL10Dialect
```

Comment:
you added datasource params too early, please remove them

Reply:
fixed db